UI/window-basic-main.cpp

```diff
@@ -4553,7 +4553,13 @@ void OBSBasic::on_scenes_customContextMenuRequested(const QPoint &pos)
 	if (item) {
 		QAction *pasteFilters =
 			new QAction(QTStr("Paste.Filters"), this);
-		pasteFilters->setEnabled(copyFiltersString);
+		OBSSource filterSource = NULL;
+		if (copyFiltersString) {
+			filterSource =
+				obs_get_source_by_name(copyFiltersString);
+			obs_source_release(filterSource);
```

Comment:
We recently added the OBSSourceAutoRelease wrapper, which avoid the need for the extra `obs_source_release()` call. As such, the wrapper should be used here.